packages/@react-aria/dnd/src/DragManager.ts

```diff
@@ -494,7 +495,16 @@ class DragSession {
 
     // Blur and re-focus the drop target so that the focus ring appears.
     if (this.currentDropTarget) {
-      this.currentDropTarget.element.blur();
+      // Since we cancel all focus events in drag sessions, refire blur to make sure state gets updated so drag target doesn't think it's still focused
+      // i.e. When you from one list to another during a drag session, we need the blur to fire on the first list after the drag.
+      if (!this.dragTarget.element.contains(this.currentDropTarget.element)) {
+        this.dragTarget.element.dispatchEvent(new FocusEvent('blur'));
+        this.dragTarget.element.dispatchEvent(new FocusEvent('focusout', {bubbles: true}));
```

Comment:
Just wanna double check, but the `blur` is strictly for the original drag target (e.g. the row) and the `focusout` is for the benefit of the Virtualizer's `onBlur`?

Reply:
Yes, exactly